.github/dependabot.yml

```diff
@@ -1,10 +1,5 @@
 version: 2
 updates:
-  - package-ecosystem: pip
```

Comment:
This PR disables dependabot for pip packages entirely. That seems wrong.

Reply:
Yes, that is wrong.